### PR TITLE
python3Packages.qpageview: init at 0.6.2

### PR DESCRIPTION
**Pull request by camillemndn**
###### Description of changes

Added qpageview. It is a necessary dependency for updating and unbreaking Frescobaldi. 

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/qpageview/default.nix [hunk 1]
```diff
@@ -0,0 +1,32 @@
+{ lib
+, python3
+, fetchFromGitHub
+, python3Packages
```
Comment (JulienMalka) [inline pkgs/development/python-modules/qpageview/default.nix:4]:
```suggestion
, python3Packages
, pythonOlder
```

File: pkgs/development/python-modules/qpageview/default.nix [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+, pythonOlder
+}:
+
+python3.pkgs.buildPythonApplication rec {
```
Comment (JulienMalka) [inline pkgs/development/python-modules/qpageview/default.nix:8]:
```suggestion
python3.pkgs.buildPythonPackage rec {
```

**Review by JulienMalka — CHANGES_REQUESTED** [review]

**Comment by JulienMalka** [comment]
Result of `nixpkgs-review pr 214007` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>python310Packages.qpageview</li>
  </ul>
</details>

**Review by JulienMalka — APPROVED** [review]
Thanks for your contribution, looks good to me :)

**Comment by wegank** [comment]
@ofborg build python3Packages.qpageview

File: pkgs/development/python-modules/qpageview/default.nix [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{ lib
+, fetchFromGitHub
+, python3
+, buildPythonPackage
+, python3Packages
+, pythonOlder
+}:
+
+python3.pkgs.buildPythonPackage rec {
```
Comment (NickCao) [inline pkgs/development/python-modules/qpageview/default.nix:9]:
```suggestion
python3Packages.buildPythonPackage rec {
```
Also remove the now unused `python3` argument above.

**Review by NickCao — APPROVED** [review]

**Comment by NickCao** [comment]
@ofborg build python3Packages.qpageview

**Pull request merged**